.github/workflows/publish_to_pypi.yml

```
@@ -0,0 +1,117 @@
+      uses: actions/download-artifact@v4
+      with:
+        name: python-package-distributions
+        path: dist/
+    - name: Verify downloaded artifacts
+      run: |
+        ls dist/*.whl dist/*.tar.gz || exit 1
+    - name: Publish distribution 📦 to PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        verbose: true
+        verbose: true
```

actionlint

key "verbose" is duplicated in "with" section. previously defined at line:65,col:9. note that this key is case insensitive

(syntax-check)